src/PanZoom.js

```diff
@@ -5,6 +5,8 @@ import type { Coordinates, BoundCoordinates, TransformationParameters, Transform
 import { TransformMatrix, getTransformedBoundingBox, getScaleMultiplier, boundCoordinates } from './maths'
 import { captureTextSelection, releaseTextSelection } from './events'
 
+const defaultNormalizeConfig = require('./config')
```

Comment:
What goes into this config?

Reply:
### What's in the config file?
At the moment, the config contains values for clamping the movement values to certain minimum amounts. So we could configure the minimum of movement a for typical pan, or we could potentially add more options to multiply the speed in certain cases. For example, we could have config options for trackpad and mouse wheel speed.

### Why have a separate config file and not just props?
We may not need a separate config file if we're okay with passing props into component. Initially, I thought  we would need the config file to make it easier to update the build artifacts without updating the rest of the code. Though that's mostly just a workaround because I've checked in the build artifacts, which is not ideal, but it was a quick hack to enable the NPM package to be installed by a git-url.

We probably can just pick one way of overriding the code, probably through props, but if we ever want to update the defaults without rebuilding the module code, we could just replace the config file with another one without causing a module rebuild.